packages/react/package.json

```diff
@@ -146,37 +119,29 @@
     "@types/jest": "^27.4.1",
     "@types/mdx": "^2.0.3",
     "@types/node": "^17.0.45",
-    "@types/react": "^17.0.11",
-    "@types/react-dom": "^17.0.20",
     "@types/video.js": "7.3.57",
+    "@vitejs/plugin-react": "^4.3.1",
     "glob": "^10.3.10",
     "http-server": "^14.1.0",
     "identity-obj-proxy": "^3.0.0",
     "jest-environment-jsdom": "27.5.1",
-    "nodemon": "^2.0.20",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-element-to-jsx-string": "^15.0.0",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
     "react-scripts": "^5.0.1",
     "remark-gfm": "^3.0.1",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-delete": "^2.0.0",
-    "rollup-plugin-livereload": "^2.0.5",
-    "rollup-plugin-multi-input": "^1.3.1",
-    "rollup-plugin-mv": "^0.0.2",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-typescript2": "^0.34.1",
     "screenfull": "^5.2.0",
     "storybook": "^8.1.9",
     "storybook-addon-rtl": "^1.0.0",
     "ts-dedent": "^2.2.0",
     "ts-jest": "^29.1.1",
     "tslib": "^2.3.1",
-    "vite": "^5.2.0"
+    "vite": "^5.2.0",
+    "vite-plugin-dts": "^3.9.1",
+    "vite-plugin-static-copy": "^1.0.6"
   },
   "peerDependencies": {
-    "react": ">16",
-    "react-dom": ">16"
+    "react": ">17",
```

Comment:
Since you upgraded to react 18, shouldn't these be changed to 18?

Reply:
`>17`  will target everything above "17th" version 

<img width="1197" alt="image" src="https://github.com/user-attachments/assets/c7976c91-63cd-47a2-806b-df4ef3e8d197">

Reply:
Sure but >17 also includes all React 17 versions. Since we are using React 18 internally, I think we should set that as our peer dependency because the older versions could behave differently.

Reply:
I will freeze it to the "18" then for readability